src/GraphQL/GraphQLMetadataAttribute.cs

```diff
@@ -107,6 +107,9 @@ public override void Modify(IGraphType graphType)
 
             if (DeprecationReason != null)
                 graphType.DeprecationReason = DeprecationReason == "" ? null : DeprecationReason;
+
+            if (graphType is IObjectGraphType objectGraphType && IsTypeOf != null)
```

Comment:
Was lost in prev PRs?

Reply:
In a manner of speaking.  This entire method is new as of the last PR.  I missed adding this in, yes.  `IsTypeOf` has always continued to work for the schema builder where it was designed for.